README.md

```diff
@@ -1,443 +1,302 @@
-# Fast Sparse Decision Tree Optimization via Reference Ensembles
+# Generalized Optimal Sparse Decision Trees
```

Comment:
This codebase is the implementation for Fast Sparse Decision Tree Optimization via Reference Ensembles. It's also where pypi points now for the currently up-to-date GOSDT code, but I think people should be citing both, especially if they're using depth constraints and threshold guessing, two of the key additions with the Fast Sparse paper. I'd like to have the readme reflect that more clearly, and at the very least to continue citing both papers, not just one (I can take a pass later for this).